tools/benchmark.py

```diff
@@ -154,6 +155,18 @@ def run_syscall_count_benchmark(deno_path):
     return syscall_count_map
 
 
+def run_max_mem_benchmark(deno_path):
+    cmd = ["/usr/bin/time", "-v", deno_path, "tests/002_hello.ts", "--reload"]
+    out = tty_capture(cmd, '')
+    for row in out[2].split('\n'):
+        if 'Maximum resident set size (kbytes)' in row or 'maximum resident set size (kbytes)' in row:
```

Comment:
Can you copy and paste an example output into tools/testdata - then demonstrate parsing it gets the correct value (break out these few lines here into a separate function) the test can go into benchmark_test.py. The strace tests in there can serve as an example.

Also rebase your code onto the latest master - it seems appveyor is failing because this is based off old commits?

Reply:
`git rebase master` does nothing

Reply:
oops I was wrong, it's not that you're based off an old branch, it's that you have checked in a directory that shouldn't be there `js/deps/https/deno.land/x/std` 

(Consider doing "git show --stat" on your commits to ensure they only include the relevant changes.)

Reply:
nit: `'maximum resident set size (kbytes)' in row.lower()`

Reply:
I've got one in one commit
![изображение](https://user-images.githubusercontent.com/19504461/55668858-0386da80-5878-11e9-9eed-db5ae4350b3c.png)